android/lib/src/main/java/com/swmansion/gesturehandler/GestureHandlerOrchestrator.java

```diff
@@ -298,7 +285,16 @@ private void deliverEventToGestureHandler(GestureHandler handler, MotionEvent ev
       return;
     }
     float[] coords = sTempCoords;
-    extractCoordsForView(handler.getView(), event, coords);
+    if (!extractCoordsForView(handler.getView(), event, coords)) {
+      if (!isFinished(handler.getState())) {
+        if (handler.getState() == GestureHandler.STATE_ACTIVE) {
+          handler.end();
```

Comment:
I don't think we ever want to `end` gesture in such a situation. Especially that we are not even able to reliably reproduce scenario in which this code gets triggered. I think the safest solution would be to follow the same code path that we normally execute when view is detached and call `cancel` on the handler -> https://github.com/kmagiera/react-native-gesture-handler/blob/master/android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerRegistry.java#L65